validation.js

```diff
@@ -0,0 +1,22 @@
+const email = document.getElementById('email');
+const emailInput = document.querySelector('small');
+
+function validateLowerChar() {
+  const EMAIL_REGEX_L = /^[a-z0-9._%+-]+@[a-z0-9.-]+.[a-z]{2,}$/;
+
+  if (email.value === '') {
+    emailInput.style.display = 'none';
+  }
+  if (!EMAIL_REGEX_L.test(email.value)) {
```

Comment:
- [ ] Great job so far. it checks the email content well if it is lower case but still submits the form succesfully after showing an error message. please note that according to the project requirement, the form should not submit if there is an uppercase letter or if validation is not ok. kindly ensure The form `"submit"` or the submit button/input `"click"` event is used to attach the function checking the email format. 

![Screenshot (953)](https://user-images.githubusercontent.com/65298048/232072999-ae08d353-b950-4ac0-ab00-0fe714e11361.png)